### PR TITLE
added option to limit logging by level in Flask

### LINKED ISSUE #342: [Flask] logging=True can be extremely noisy, needs to be documented
Using `ElasticAPM(logging=True)` sets up our `LoggingHandler` to log *everything* to ElasticAPM. This is almost always not what the user wants, and needs to be documented as possibly very noisy.

### LINKED ISSUE #343: [Flask] Add option to set up logging by log level
Currently, instantiating the flask client with `logging=True` will log everything to Elastic APM. It would be nice to also support `logging=SOME_LOG_LEVEL`, which would only send messages with level `SOME_LOG_LEVEL` and higher to Elastic APM.

### PR DESCRIPTION
**Pull request by beniwohli**
fixes elastic/apm-agent-python#342
fixes elastic/apm-agent-python#343

### CODE REVIEW

File: docs/flask.asciidoc [hunk 1]
```diff
@@ -143,6 +143,18 @@ from elasticapm.contrib.flask import ElasticAPM
 apm = ElasticAPM(app, logging=True)
```
Comment (roncohen) [inline docs/flask.asciidoc]: `logging=True` seems like a footgun. I'd rather lead with the explanation of `logging=logging.ERROR` and then mention that you can do `logging=True` to get everything incl. the nice warning that you currently have below the `logging=logging.ERROR` example.

It's also worth considering to completely stop mentioning `logging=True`. Setting `logging=logging.DEBUG` has the same effect and is more obviously a footgun.

Comment (beniwohli) [reply docs/flask.asciidoc]: That makes thev docs ten times better! 👍

**Comment by beniwohli** [comment]
Looks like adding `logging=xyz` has some side effects, not sure which. I hoped that removing the installed handler in the fixture tear down would be enough, but that doesn't seem to be the case :(

**Comment by beniwohli** [comment]
merged in e05afc43987

**Pull request closed without merging**